lib/model/QubitAccessLog.php

```diff
@@ -48,4 +48,19 @@ public static function getPopularThisWeek(array $options = [])
 
         return $stmt->fetchAll();
     }
+
+    /**
+     * Delete old access_log records.
+     *
```

Comment:
Nitpick: omment doesn't document the `$expiryData` argument using `@param`.

Reply:
Good call.  I've added a comment for the `$expiryDate` parameter.